src/components/Detail/UpdateForm.tsx

```diff
@@ -0,0 +1,76 @@
+import { useState } from 'react';
+
+import { trpc } from '../../utils/trpc';
+import { Button } from '../Button/Button';
+import { TextField } from '../TextField/TextField';
+import style from './UpdateForm.module.scss';
+
+type UpdateFormProps = {
+  activity: {
+    name: string;
+    amount: number;
+    unit: string;
+    timesDone: number;
+    id: number;
+  };
+};
+
+export const UpdateForm: React.FC<UpdateFormProps> = ({ activity }) => {
+  const [currentAmount, setCurrentAmount] = useState(0);
```

Comment:
Same as above. I think this could be explicitly handled as registration form.

Reply:
same as above needs to be resolved.